Makefile

```
@@ -147,10 +147,18 @@ help:
 	fi
 	@echo
 
+### check-rust : check if Rust is installed in the environment
+.PHONY: check-rust
+check-rust:
+	@if [ ! which rustc >/dev/null ]; then \
```

```suggestion
	@if ! [ $(shell command -v rustc) ]; then \
```
proposed by @leslie-tsang